org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/desktop.css.dsp

```diff
@@ -266,6 +266,10 @@
 	padding: 8px;
 }
 
+.help-entitytype {
```

Comment:
when changing theme please change also the timestamp on file org.adempiere.ui.zk/WEB-INF/src/metainfo/zk/lang-addon.xml
as explained at https://wiki.idempiere.org/en/Check_List_Peer_Review#Code_Peer_Review